.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: release
+
+permissions:
+  contents: write
+  packages: write
+  statuses: write
+  pull-requests: read
```

Comment:
The goreleaser docs don't mention needing either of these permissions? https://goreleaser.com/ci/actions/#token-permissions